x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/XPackPluginTests.java

```diff
@@ -82,8 +97,42 @@ public void testNodesNotReadyForXPackCustomMetadata() {
         }
     }
 
-    private XPackPlugin createXPackPlugin(Settings settings) throws Exception {
-        return new XPackPlugin(settings, null){
+    public void testCustomRestWrapperDeprecationMessage() throws Exception {
```

Comment:
I don't think this test belongs here.
It should be in `ActionModuleTests`.

Reply:
Moved